base-notebook/Dockerfile

```diff
@@ -52,9 +52,8 @@ RUN useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
 
 USER $NB_USER
 
-# Setup jovyan home directory
-RUN mkdir /home/$NB_USER/work && \
-    echo "cacert=/etc/ssl/certs/ca-certificates.crt" > /home/$NB_USER/.curlrc
+# Setup work directory for backward-compatibility
+RUN mkdir /home/$NB_USER/work
```

Comment:
> Is there anything else in docker-stacks using this

No, this directory is deliberately created and left empty, so that it might be mounted as a volume. Nothing is done with it in docker-stacks.

> is this of concern for some known downstream use cases?


juptyerhub-deploy-docker [uses this](https://github.com/jupyterhub/jupyterhub-deploy-docker/blob/master/jupyterhub_config.py#L34) by default, but since it mounts it as a volume, it probably wouldn't be affected. We can possibly get away with removing the directory without harming anything I *know of*, at least.